server/routes/search/searchController.ts

```diff
@@ -124,7 +126,7 @@ export default class SearchController {
     req.session.searchParams.filters?.forEach(entry => {
       searchParams.append('filters', entry)
     })
-    return searchParams ? '' : searchParams.toString()
+    return searchParams ? searchParams.toString() : ''
   }
```

Comment:
I guess you could just do

`return searchParams?toString() ?? '' `

Reply:
will add in next PR